src/sentry/services/hybrid_cloud/rpc.py

```diff
@@ -350,6 +369,25 @@ def create_delegation(cls, use_test_client: bool | None = None) -> Self:
         return service  # type: ignore[return-value]
 
 
+def list_all_service_method_signatures() -> Iterable[RpcMethodSignature]:
+    """List signatures of all RPC methods in the global registry."""
+
+    yielded: set[DelegatingRpcService] = set()
+    while True:
+        # Kludge around new services being registered as a side effect of import
+        # statements during iteration.
+        #
+        # TODO: This suggests we can't rely on the registry being fully populated at
+        #  the time this method is called, even with the kludge. Investigate?
```

Comment:
it might make sense to use [`pkgutil.walk_packages`](https://docs.python.org/3/library/pkgutil.html#pkgutil.walk_packages) to walk all the rpc modules to trigger their import side-effects and then this should be more straightforward?

Reply:
👍 to `pkgutil.walk_packages`.

For the record, I would definitely consider solving this in a non-kludgey way to be a blocker to production usage. The only reason I don't have this PR in draft status is because we aren't ready to invoke this command anyway, except with the `--partial` flag in dev contexts.

Reply:
Took me a while before I circled back to this, but you may consider it un-kludged. https://github.com/getsentry/sentry/pull/67175/commits/f6b532547e5c028a5f0b23f9bcb3151f2be68df4